cluster_intro.markdown

```diff
@@ -0,0 +1,88 @@
+---
```

Comment:
if you want to create multiple pages it'd be best to create separate branches for each page and contribute them separately!